platform/broadcom/sonic-platform-modules-inventec/d7332/modules/i2c-mux-pca9541.c

```diff
@@ -0,0 +1,573 @@
+/*
+ * I2C multiplexer driver for PCA9541 bus master selector
+ *
+ * Copyright (c) 2010 Ericsson AB.
+ *
+ * Author: Guenter Roeck <linux@roeck-us.net>
+ *
+ * Derived from:
+ *  pca954x.c
+ *
+ *  Copyright (c) 2008-2009 Rodolfo Giometti <giometti@linux.it>
+ *  Copyright (c) 2008-2009 Eurotech S.p.A. <info@eurotech.it>
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2. This program is licensed "as is" without any
```

Comment:
i2c-mux-pca9541 is in linux kernel, why do we need this one?

Reply:
To add pca9641 in it to fulfill customer requirement

![image](https://user-images.githubusercontent.com/44458761/67060489-98578280-f18f-11e9-9cb9-314e55b93348.png)